src/server/database/SQLite.ts

```
@@ -4,8 +4,16 @@ import {GameOptions} from '../game/GameOptions';
 import {GameId, ParticipantId} from '../../common/Types';
 import {SerializedGame} from '../SerializedGame';
 
-import sqlite3 = require('sqlite3');
-import {RunResult} from 'sqlite3';
+import * as sqlite3 from 'sqlite3';
+
+let Database: typeof sqlite3.Database | undefined;
+
+try {
```

Why not let this throw?